source/documentation/query/lateral-join.md

```diff
@@ -131,11 +131,15 @@ available to "pattern". This is similar to an SQL
 
 `LATERAL` is added to the SPARQL grammar at rule `[[56] GraphPatternNotTriples](https://www.w3.org/TR/sparql11-query/#rGraphPatternNotTriples)`. As a syntax form, it is similar to `OPTIONAL`.
 
-```ebnf
-[56]    GraphPatternNotTriples    ::=    GroupOrUnionGraphPattern | OptionalGraphPattern | LateralGraphPattern | ...
-[57]    OptionalGraphPattern      ::=    'OPTIONAL' GroupGraphPattern
-[  ]    LateralGraphPattern       ::=    'LATERAL' GroupGraphPattern
-```
+<div class="font-monospace">
```

Comment:
We only need to use this div + font-monospace class for tables that we want to format as code.

Any code block with pre>code, or with the three backticks, will now automatically have the `font-monospace` inherited/included, and the size was also set to `1rem` as it was looking a bit smaller than the rest of the text. Happy to drop the font-size if the other value was better.